releases/cosmwasm/2025-02-xrplevm-GMP-v6.4.0.md

```diff
@@ -0,0 +1,351 @@
+# Ripple EVM Sidechain GMP v6.4.0
+
+|                | **Owner**                                                                   |
+| -------------- | --------------------------------------------------------------------------- |
+| **Created By** | @blockchainguyy <ayush@interoplabs.io>                                      |
+| **Deployment** | @blockchainguyy <ayush@interoplabs.io>, @talalashraf <talal@interoplabs.io> |
+
+| **Network**          | **Deployment Status** | **Date** |
+| -------------------- | --------------------- | -------- |
+| **Devnet Amplifier** | -                     | TBD      |
+| **Stagenet**         | -                     | TBD      |
```

Comment:
We can say the staging environment that we will deploy to testnet will be `stagenet` for this purpose

Reply:
That's only temporary, we'll be adding the proper stagenet connection for long term support. We are skipping it temporarily to get the staging connection on testnet. We can update this doc before the official testnet deployment